### DISCUSSION TITLE
Add a Workflow for Wiz vulnerabilities

### DISCUSSION THREAD

**Discussion by IH4T31BM** [IBM/IBM-QRadar-Universal-Cloud-REST-API#253] [Ideas] [answered]
Is it possible to add a Workflow to pull Wiz vulnerabilities? aside from wiz issues.

**Comment by ChrisCollinsIBM** [comment]
Are you looking to get the vulnerabilities into the asset model in QRadar from Wiz? Or just get some type of event generated if new vulnerabilities are found on a host?

The asset APIs for QRadar are currently a bit limited but undergoing some planning for improvements in the near future, and once that happens you could definitely have a UREST workflow that posts assets/vulns instead of events, but at this time those APIs are not in place.

If you're looking for events to be triggered related to vulnerabilities maybe the Wiz contributors @suraj-metron or @solalraveh could offer some info.

**Comment by IH4T31BM** [comment]
Yes, I'm looking to get regular events when new vulnerabilities are found on a host.
Thanks for connecting me with @suraj-metron  and @solalraveh :)

**Comment by yanivblumWiz** [comment, marked as answer]
@IH4T31BM 
Can you please submit this request via your Wiz Account Manager so we can properly track and handle this request?
